index.js

```diff
@@ -43,28 +44,19 @@ for (var i = 0; i < filesLoadedBeforeTrace.length; i++) {
 var onUncaughtExceptionValues = ['ignore', 'flush', 'flushAndExit'];
 
 /**
- * Phantom implementation of the trace agent. This allows API users to decouple
+ * Phantom implementation of the trace api. This allows API users to decouple
  * the enable/disable logic from the calls to the tracing API. The phantom API
  * has a lower overhead than isEnabled checks inside the API functions.
  * @private
  */
 var phantomTraceAgent = {
-  startSpan: function() { return SpanData.nullSpan; },
-  endSpan: function(spanData) { spanData.close(); },
-  runInSpan: function(name, labels, fn) {
-    if (typeof(labels) === 'function') {
-      fn = labels;
-    }
-    fn(function() {});
-  },
-  runInRootSpan: function(name, labels, fn) {
-    if (typeof(labels) === 'function') {
-      fn = labels;
-    }
-    fn(function() {});
-  },
-  setTransactionName: function() {},
-  addTransactionLabel: function() {}
+  enhancedDatabaseReportingEnabled: function() { return false; },
+  runInRootSpan: function(opts, fn) { return fn(PluginApi.nullSpan); },
+  createChildSpan: function(opts) { return PluginApi.nullSpan; },
+  wrap: function(fn) { return fn; },
+  wrapEmitter: function(ee) { return ee; },
+  constants: constants,
+  labels: TraceLabels
 };
 
 /** @private */
```

Comment:
Add JSDoc that this would either be a PhantomAgent or a PluginAPI object. Also, the name `agent` seems wrong when this can be an object of the type `PluginAPI`. I am guessing this is going to be cleaned up in a follow-on?